modules/apm/src/main/java/org/elasticsearch/telemetry/apm/internal/APMAgentSettings.java

```diff
@@ -44,7 +44,6 @@ public void addClusterSettingsListeners(ClusterService clusterService, APMTeleme
 
         clusterSettings.addSettingsUpdateConsumer(TELEMETRY_TRACING_ENABLED_SETTING, enabled -> {
             apmTracer.setEnabled(enabled);
-            this.setAgentSetting("instrument", Boolean.toString(enabled));
```

Comment:
This was accidentally enabled in https://github.com/elastic/elasticsearch/pull/99832/files#diff-8e5ca1315006cc00c58ef42d4bfc14f50b30d11567a845f8e551ecf3e52c830bR58, though it's not required for recording manual traces.